docs/destinations/forwarding-events/snowbridge/testing/index.md

```diff
@@ -98,6 +98,31 @@ transform {
 }
 ```
 
+### Adding a http request template
```

Comment:
http -> HTTP

Reply:
Thanks for flagging apologies, I'll admit I forgot to check it against the style guide.

I've given it a once over since and didn't see any further misses on that front but lmk if you spot any more!